### PR TITLE
Add fake app to suite object for testings

### PR DESCRIPTION
**Pull request by prestwich**

### CODE REVIEW

File: golang/fake_app.go [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+	stakingKeeper := staking.NewKeeper(
+		app.cdc,
+		keys[staking.StoreKey],
+		tkeys[staking.TStoreKey],
+		app.supplyKeeper,
+		stakingSubspace,
+		staking.DefaultCodespace,
+	)
+
+	app.distrKeeper = distr.NewKeeper(
```
Comment (ErinHales) [inline golang/fake_app.go]: Missing docs

File: golang/fake_app.go [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+		keys[distr.StoreKey],
+		distrSubspace,
+		&stakingKeeper,
+		app.supplyKeeper,
+		distr.DefaultCodespace,
+		auth.FeeCollectorName,
+		app.ModuleAccountAddrs(),
+	)
+
+	app.slashingKeeper = slashing.NewKeeper(
```
Comment (ErinHales) [inline golang/fake_app.go]: Missing docs

Comment (ErinHales) [reply golang/fake_app.go]: Several missing throughout this file actually

Comment (prestwich) [reply golang/fake_app.go]: wait what needs docs here?

Comment (ErinHales) [reply golang/fake_app.go]: Oh yeah, sorry. This was a bad place to put the comment.  Just look through this file and make sure everything has docs.

Comment (ErinHales) [reply golang/fake_app.go]: Oh wait, this might not need a doc, but there were for sure functions later in this file that were missing docs.

**Pull request merged**